js/src/components/merchant-center-select-control/index.js

```diff
@@ -27,12 +37,22 @@ const MerchantCenterSelectControl = ( props ) => {
 	options.sort( ( a, b ) => {
 		return a.label.localeCompare( b.label );
 	} );
-	options.unshift( {
-		value: '',
-		label: __( 'Select one', 'google-listings-and-ads' ),
-	} );
 
-	return <AppSelectControl options={ options } { ...props } />;
+	useEffect( () => {
+		// Triggers the onChange event in order to pre-select the initial value
+		if ( value === undefined ) {
+			onChange( options[ 0 ].value );
```

Comment:
I know that when the options are empty this component is not rendered, but we could change it to
```suggestion
			onChange( options[ 0 ]?.value );
```

So if this component is reused and the options are empty, the component will not break.